x-pack/solutions/security/packages/features/src/security/product_feature_config.ts

```diff
@@ -111,6 +111,11 @@ export const securityDefaultProductFeaturesConfig: DefaultSecurityProductFeature
     },
   },
 
+  // Endpoint Essentials/Complete tiers
```

Comment:
This file should be tier-agnostic; this information is already in the `pli_config.ts`

Reply:
Got it! I'll remove the comment